roles/eda/templates/eda-activation-worker.deployment.yaml.j2

```diff
@@ -66,6 +66,50 @@ spec:
 {% endif %}
 {% if bundle_ca_crt  %}
       initContainers:
+      - name: wait-for-migrations
```

Comment:
I think we need to move this (including the `initContainers` statement) outside of the bundle_ca_crt condition otherwise this won't work on default deployment

Reply:
Good spot, I just pushed a change to move it outside of the `bundle_ca_crt` block.  Thanks!